docs/content/doc/usage/issue-pull-request-templates.en-us.md

```diff
@@ -267,3 +279,17 @@ For each value in the options array, you can set the following keys.
 |----------|------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|---------|---------|
 | label    | The identifier for the option, which is displayed in the form. Markdown is supported for bold or italic text formatting, and hyperlinks. | Required | String  | -       | -       |
 | required | Prevents form submission until element is completed.                                                                                     | Optional | Boolean | false   | -       |
+
+## Syntax for issue config
+
+This is a example for a issue config file
+
+```yaml
+BlankIssuesEnabled: true
+```
+
+### Possible Options
+
+| Key                | Description                                                                                           | Type    |
+|--------------------|-------------------------------------------------------------------------------------------------------|---------|
+| BalnkIssuesEnabled | If set to false, the User is forced to use a Template                                                 | Boolean |
```

Comment:
```suggestion
| BlankIssuesEnabled | If set to false, the User is forced to use a Template                                                 | Boolean |
```

Reply:
Also, on GitHub, this is written in snake case: `blank_issues_enabled: false`.
Does our YAML parser parse all text cases, or do we need to adapt something?
And if it allows multiple text cases, we should probably document that.

Reply:
It should be `blank_issues_enabled`.

The [definition](https://github.com/go-gitea/gitea/pull/20956/files#diff-828ef82cb24a01a8bb0409e2bbb87296e058c1c3e78d45d599e800268ee7d2b8R194) named it with tag `yaml:"blank_issues_enabled"`.

Reply:
It is `blank_issues_enabled`. This is a mistake of me while writing the documentation. I will fix it in the next commit.